packages/hmr/package.json

```diff
@@ -0,0 +1,68 @@
+{
+	"name": "@sapphire/plugin-hmr",
+	"version": "0.0.1",
+	"description": "Plugin for @sapphire/framework for hot module replacement with commands, listeners and more",
```

Comment:
```suggestion
	"description": "Plugin for @sapphire/framework for hot module replacement for pieces",
```

Reply:
I'd argue it should be reload not replacement